### PR TITLE
Add v0.6.2 release notes

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: website/blog/2020-06-21-radium.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Thanks to an awesome contribution from @tanishiking we are now able to provide
+you with information about usages if 'Goto definition' was used on the
+definition itself.
+
+![Fallback](https://i.imgur.com/BT3h0FJ.gif)
+
+## Support for Scala 2.13
+
+A lot of work was done by @gabro to make Metals compatible with Scala 2.13. More
```
Comment (tgodzik) [inline website/blog/2020-06-21-radium.md]: @gabro Anything you would like to add? That was a lot of work and I feel I am simplying it too much.

Also used what you wrote on gitter about dropping older Scala versions - I think it was pretty concise and well explained the situation.

File: website/blog/2020-06-21-radium.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+- digest only `.sbt` files at workspace level for sbt projects (thanks
+  @wojciechUrbanski!)
+- fix SemanticDB plugin path on window for Gradle workspaces
+- update default Mill version to 0.4.1 to fix an issue with classpath not
+  containing Scala library
+- Metals now pick up the version from `.mill-version` for `millw` script
+
+## Metals now works with JDK 11
+
+Thanks to the joint effort by @olafurpg and @er1c Metals now works with JDK 11! The effort included fixing several JDK 11 issues and some awesome improvements to the indexing itself. <TODO>
```
Comment (tgodzik) [inline website/blog/2020-06-21-radium.md]: @olafurpg It's a bit short now - anything you want to add?

File: website/blog/2020-06-21-radium.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+`2.12.8` and `2.13.0` while deprecating `2.11.12` and `2.12.7`. By deprecating `2.11.12` we want to push users to upgrade if they can as it is hard for Metals to
+support 2.11 with the same levels of features as more recent versions. This come up when making Metals compatible with Scala 2.13 and it turns out it is getting very difficult for us to support multiple Scala versions. It must
+be noted that we're not dropping `2.11.12` anytime soon and even when we will,
+2.11 users will always be able to use an older version of Metals.
+
+# Fixes for auto importing builds
+
+- digest only `.sbt` files at workspace level for sbt projects (thanks
+  @wojciechUrbanski!)
+- fix SemanticDB plugin path on window for Gradle workspaces
```
Comment (ayoub-benali) [inline website/blog/2020-06-21-radium.md]: window**s**

File: website/blog/2020-06-21-radium.md (hunk 2 above)
Comment (olafurpg) [reply website/blog/2020-06-21-radium.md]: Done.

File: website/blog/2020-06-21-radium.md (hunk 1 above)
Comment (olafurpg) [reply website/blog/2020-06-21-radium.md]: I added a more detailed explanation of both what's missing in 2.13 support and also added details about 2.11 deprecation.

File: website/blog/2019-06-24-radium.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+## Miscellaneous fixes
+
+- fix off-by-one for parameter hints when the cursor was after the closing `)`
+  parenthesis
+- make sure we add an autoimport in the correct line in case of brackets
+- document symbol outline no longer fails when `val _ = ()` is the only thing in
+  a block expression
+- nested objects are now imported correctly in case of deeper nesting
+- fix concurrent modification in text document cache (thanks
```
Comment (gabro) [inline website/blog/2019-06-24-radium.md]: we also miss the commit character removal PR

**Comment by tgodzik** [comment]
Closing since we are thinking about more major release

**Pull request closed without merging**